test/Microsoft.IdentityModel.AotCompatibility.Tests/Microsoft.IdentityModel.AotCompatibility.Tests.csproj

```diff
@@ -5,6 +5,7 @@
   <PropertyGroup>
     <!-- This test only needs to run on .NET -->
     <TargetFrameworks>net6.0</TargetFrameworks>
+    <LangVersion>10</LangVersion>
```

Comment:
nit: should we keep this consistent with commonTest.props by using 11?

Reply:
We would, but this does not work with net6.0

Reply:
We could also remove the targetframeworks and this line

Reply:
I see, resolving this comment. Thanks Jean-Marc!